### PR TITLE
docs: improve "running from CI builds" documentation

### PR DESCRIPTION
**Pull request by GaryElshaw**
Minor amendments for those unfamiliar with using the macOS Terminal.

### CODE REVIEW

**Comment by Coeur** [comment]
Hum... too baby level for this project. If one doesn't know how to copy-paste a line in Terminal, they would hardly be the type to register on GitHub, post an issue which is edge case and non-reproducible by others, requiring their verification from a CI build before merging a potential fix...

File: docs/Testing-Transmission.md [hunk 1]
```diff
@@ -6,11 +6,20 @@ Download the desired "from-tarball" Artifact at the bottom of the page.
...
-You need to mark Transmission.app both as executable and not quarantined:
+You need to mark Transmission.app both as executable and not quarantined using the Terminal application (in Applications/Utilities):
 
 1. `chmod +x Transmission.app/Contents/MacOS/Transmission`
+
+   If you're unfamiliar with using the Terminal application, the following steps might be of help.
+
+   1. Open the Terminal application and type: chmod +x (and add a space after the x).
+   2. Right-click or Ctrl-click the Transmission application and select 'Show Package Contents'.
+   3. Navigate to the 'MacOS' directory; drag and drop 'Transmission' on to the Terminal window you had opened in step 1 and press Return.
```
Comment (Coeur) [inline docs/Testing-Transmission.md:17]: This is over complicated and unnecessary.

Let's just add "_using the Terminal application, go to the folder containing Transmission.app and copy-paste those commands_".

**Review by Coeur — CHANGES_REQUESTED** [review]
The drag&drop part is not a necessary step: copy-pasting the commands as they are is enough.

**Comment by GaryElshaw** [comment]
No worries.

**Pull request closed without merging**